core/workspace_svg.ts

```diff
@@ -1327,10 +1327,6 @@ export class WorkspaceSvg
       .flatMap((block) => block.getDescendants(false))
       .filter((block) => block.isInsertionMarker())
       .forEach((block) => block.queueRender());
-
-    renderManagement
-      .finishQueuedRenders()
```

Comment:
Should `render` no longer call `renderManagement.finishQueuedRenders()` at all?

(Maybe the queued renderers will be finished automatically, and the method call is not needed if you don't need the resulting promise?)